cmd/trace-agent/run.go

```diff
@@ -48,6 +48,9 @@ const messageAgentDisabled = `trace-agent not enabled. Set the environment varia
 DD_APM_ENABLED=true or add "apm_config.enabled: true" entry
 to your datadog.yaml. Exiting...`
 
+// Stack depth of 3 since the `corelogger` struct adds a layer above the logger
+const stackDepth = 3
```

Comment:
Are you sure it's 3 ?
I did a quick test and I'm thinking it should be 2 but I didn't actually run the trace-agent so I might be wrong

Reply:
Yeah, unit and manual testing both say it's 3. I was also confused when I saw it :/